src/main/java/net/william278/velocitab/config/Settings.java

```diff
@@ -59,6 +59,10 @@ public class Settings {
     @YamlKey("formats")
     private Map<String, String> formats = Map.of("default", "&7[%server%] &f%prefix%%username%");
 
+    @YamlKey("nametags")
+    @YamlComment("Nametag(s) to display above players' heads for each server group. To disable, set to empty")
+    private Map<String, String> nametags = Map.of("default", "&f%prefix%%username%&f%suffix%");
+
```

Comment:
I think this feature should be disabled by default -- I see it more as a feature _in addition_ to the TAB stuff, and it has limitations that the TAB system does not. :).
we'll do some documentation on how to turn it on, though!